{
  "uid" : "218d356afd475337",
  "name" : "User manually selects a different language -- @1.2 ",
  "fullName" : "Language Selection and Translation: User manually selects a different language",
  "historyId" : "b907570cd79df11b89f0177d1c526184",
  "time" : {
    "start" : 1746711076952,
    "stop" : 1746711076986,
    "duration" : 34
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: Given the system supports the following languages\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_012_steps.py\", line 14, in step_given_supported_languages\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: Given the system supports the following languages\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_012_steps.py\", line 14, in step_given_supported_languages\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the system supports the following languages:",
      "time" : {
        "start" : 1746711076954,
        "stop" : 1746711076955,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: Given the system supports the following languages\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_012_steps.py\", line 14, in step_given_supported_languages\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "4d37859f3b5b2057",
        "name" : ".table",
        "source" : "4d37859f3b5b2057.csv",
        "type" : "text/csv",
        "size" : 31
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Given a logged-in user",
      "time" : {
        "start" : 1746711076986,
        "stop" : 1746711076986,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the user selects German from the language settings",
      "time" : {
        "start" : 1746711076986,
        "stop" : 1746711076986,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then the system should display the UI in German",
      "time" : {
        "start" : 1746711076986,
        "stop" : 1746711076986,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Language Selection and Translation"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "language",
    "value" : "German"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "debff9f56d7e97e5",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/21/#testresult/debff9f56d7e97e5",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given the system supports the following languages\n",
        "time" : {
          "start" : 1746710242654,
          "stop" : 1746710242687,
          "duration" : 33
        }
      }, {
        "uid" : "af43182b4bb02a6a",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/20/#testresult/af43182b4bb02a6a",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: Given the system supports the following languages\n",
        "time" : {
          "start" : 1746709788772,
          "stop" : 1746709788806,
          "duration" : 34
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "218d356afd475337.json",
  "parameterValues" : [ "German" ]
}